domain/machine/watcher_test.go

```diff
@@ -69,5 +69,10 @@ func (s *watcherSuite) machineServiceAndWatcher(c *gc.C) (*service.WatchableServ
 	watcher, err := machineService.WatchMachines(context.Background())
 	c.Assert(err, gc.IsNil)
 	watcherC := watchertest.NewStringsWatcherC(c, watcher)
+	// Initial event.
+	watcherC.AssertOneChange()
+	// Ensure the change stream is idle.
```

Comment:
I can understand the comment for `AssertOneChange` being for the initial event, but the comment for `AssertChangeStreamIdle` adds nothing.

Reply:
Fixed.